MauldinEconomics/mautic
# New Asset filters

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? |  x
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Adds asset multi selector support. 
Adds asset filter.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1.  
2. 

#### Steps to test this PR:
1. Create a  asset.
2. Download the asset with a lead.
3. Create segment selecting the created asset.
3. Update the segment and check if there lead is in the segment.

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
